src/Module.zig

```diff
@@ -766,6 +766,13 @@ pub const Scope = struct {
         inlining: ?*Inlining,
         is_comptime: bool,
 
+        shared: *Shared,
+
+        const Shared = struct {
+            branch_count: u32 = 0,
```

Comment:
This one should be in the Inlining.Shared struct however branch_quota indeed should be shared among the analysis for an entire Decl.

Reply:
This makes it not able to be used outside an inline function.